lib/mix/tasks/ecto.gen.migration.ex

```diff
@@ -61,7 +61,6 @@ defmodule Mix.Tasks.Ecto.Gen.Migration do
 
   @impl true
   def run(args) do
-    no_umbrella!("ecto.gen.migration")
```

Comment:
We still want to raise if --migrations-path is given and there are multiple repos, no?

Reply:
Ignore me!